docs/conf.py

```diff
@@ -198,7 +199,7 @@
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-html_title = "{0} v{1}".format(project, release)
+html_title = "" # "{0} v{1}".format(project, release)
```

Comment:
```suggestion
html_title = project # "{0} v{1}".format(project, release)
```
This will keep page titles (that are shown when you hover over tab) meaningful. For e.g. With this it will be "Installation - TARDIS" instead of "Installation" for https://epassaro.github.io/tardis/branch/simple-version-docs/installation.html

Reply:
This could be meaningful in the sense that we are keeping the `project` variable as the HTML Title